operator-framework/src/main/java/com/github/containersolutions/operator/processing/EventDispatcher.java

```diff
@@ -100,14 +116,21 @@ private boolean markedForDeletion(CustomResource resource) {
     }
 
     // created to support unit testing
```

Comment:
This comment seems bit misleading as this class in invoked from regular flow too.

Reply:
> This comment seems bit misleading as this class in invoked from regular flow too.

But its still true, and this is the only reason we are created this class. This is intentional.